test/e2e/pull_test.go

```diff
@@ -62,6 +62,58 @@ var _ = Describe("Podman pull", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman pull and run on split imagestore", func() {
+		SkipIfRemote("podman-remote does not support setting external imagestore")
+
+		tmpDir := filepath.Join(podmanTest.TempDir, "splitstore")
+		podmanHello := "quay.io/podman/hello"
+
+		// pull to splitstore
+		session := podmanTest.Podman([]string{"pull", "--imagestore", tmpDir, podmanHello})
```

Comment:
Test is actually to verify the pull behavior since that was the original use-case, I chose `quay.io/podman/hello` since its small and src is maintained by `podman` maintainers itself.

Reply:
I don't think it matters whether `load` or `pull` are used.  The important part is that the destination is `containers-storage` on the `--imagestore`.

Reply:
yes I agree, let's avoid pulling an image even if it is small

Reply:
Sure let me amend the test.

Reply:
Nvm I think `podmanTest.AddImageToRWStore(ALPINE)` adding to write-able store should do the trick.